src/components/FreelancerCard.astro

```diff
@@ -78,21 +88,34 @@ const freelancerImage: ImageMetadata | undefined = imageFiles[imagePath];
         {`${freelancer.firstName} ${freelancer.lastName}`}
       </h2>
     </a>    <p class="text-pink-500 dark:text-pink-400 mb-2">{freelancer.headline}</p>
-    <p class="text-sm text-neutral-500 dark:text-neutral-500 mb-2">
-      📍 {freelancer.location} {getFlagEmoji(freelancer.countryCode)}
+    <p class="text-sm text-neutral-500 dark:text-neutral-500 mb-2 flex items-center gap-2">
+      📍 {freelancer.location} 
+      {flagImage && (
+        <Image 
+          src={flagImage}
+          alt={`Flag of ${freelancer.location}`}
+          width={20}
+          height={15}
+          class="inline-block"
+        />
+      )}
     </p>
     <p class="text-sm text-neutral-500 dark:text-neutral-500 mb-2">🗣️ {freelancer.language}</p>
     <p class="text-sm text-neutral-500 dark:text-neutral-500 mb-6">Availability: {freelancer.availability}</p>
     <div class="flex items-center mb-4 mt-4 gap-2">
       <CQIcon profileUrl={freelancer.forumProfile} size="sm" />
       {freelancer.linkedIn && (
         <a href={freelancer.linkedIn} target="_blank" rel="noopener noreferrer" class="text-neutral-800 dark:text-neutral-100 inline-flex items-center">
-          <Icon name="mdi:linkedin" class="w-6 h-6 mr-2" /> 
+          <svg class="w-6 h-6 mr-2" viewBox="0 0 24 24">
```

Comment:
what is that? this is not intuitive when you read the code - the previous one was much more readable. Why is the direct use of the path needed here? any alternatives?